Base/usr/share/man/man8/TelnetServer.md

```diff
@@ -0,0 +1,25 @@
+## Name
+
+TelnetServer - Serenity telnet server
+
+## Synopsis
+
+```**sh
+$ TelnetServer [options]
```

Comment:
The `TelnetServer` is supposed to be run as root (unless you're using a custom port), correct? Then let's make the prompt a hash sign instead of a dollar.